app/models/friendship.rb

```diff
@@ -0,0 +1,6 @@
+class Friendship < ApplicationRecord
+  belongs_to :user
+  belongs_to :friend, class_name: 'User'
+
+  validates :user_id, uniqueness: { scope: :friend_id }
```

Comment:
`{ user_id: 1, friend_id: 2 }`  is existing in database.
So what if data like `{ user_id: 2, friend_id: 1 }` wanna create ? , Is it valid ?

Reply:
when 2 users add friend, 2 records of friendship are create. So I think it enough :bowing_man: